Resources/Prototypes/_EE/Entities/Clothing/SolAlliance/SAN/Modsuits/CSA-85x.yml

```
@@ -0,0 +1,192 @@
+      key: enum.HandheldRadioUiKey.Key
+    - type: UserInterface
+      interfaces:
+        enum.HandheldRadioUiKey.Key:
+          type: HandheldRadioBoundUserInterface
+
+- type: entity
+  parent: [ClothingModsuitChestplateStandard, SolAllianceInfo]
+  id: ClothingModsuitChestplateSANMarine
+  name: CSA-85x "Mazu" modsuit chestplate
+  description: A special modular suit space-proof cover designed for compact folding inside modular suit control.
+  categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)